src/components/shared/C3Chart/Colors.js

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
What do you think about the filename put something like this`C3Chart.colors.js` ?

Reply:
I'm not so sure about this, perhaps it could move this file colors directly into shared folders, and then be reused in other components.

Reply:
But I'm not so sure if it is logical to do this, the idea of component 😕